en/docs/security-announcements/incident-clarifications/2026/axios-npm-supply-chain-compromise.md

```
@@ -34,4 +35,4 @@ For customized deployments, WSO2 recommends pinning the Axios version in package
 Based on this analysis, WSO2 products and services deployed using unmodified official release artifacts are not affected by this supply chain attack. Customers using unmodified official release artifacts are likewise not impacted.
 
 ### References
-[^1]: [https://www.stepsecurity.io/blog/axios-compromised-on-npm-malicious-versions-drop-remote-access-trojan](https://www.stepsecurity.io/blog/axios-compromised-on-npm-malicious-versions-drop-remote-access-trojan)
\ No newline at end of file
+[^1]: [https://www.stepsecurity.io/blog/axios-compromised-on-npm-malicious-versions-drop-remote-access-trojan](https://www.stepsecurity.io/blog/axios-compromised-on-npm-malicious-versions-drop-remote-access-trojan)
```

markdownlint-cli2 (0.22.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "^1"

(MD053, link-image-reference-definitions)